crates/miden-tx/Cargo.toml

```diff
@@ -25,7 +25,7 @@ testing = ["miden-objects/testing", "miden-lib/testing", "vm-processor/testing",
 
 [dependencies]
 async-trait = "0.1"
-miden-lib = { workspace = true }
+miden-lib = { workspace = true, features = ["with-debug-info"]}
```

Comment:
Should this feature be conditional? e.g., Should we add `with-debug-info` feature to this crate as well?

Reply:
I accidentally added this and removed it.

I think we don't have to add this feature to miden-tx, as we don't re-export miden-lib from miden-tx. If someone uses miden-lib they will include it directly in their `Cargo.toml` and then they can choose to enable it or not.